src/test/java/org/broadinstitute/hellbender/utils/reference/ReferenceUtilsUnitTest.java

```diff
@@ -52,6 +53,25 @@ public void testLoadFastaDictionaryFromStream() throws IOException {
         }
     }
 
+    @Test
+    public void testLoadFastaDictionaryWithFastaFile() throws IOException {
+
+        File testFastaFile = new File(hg19MiniReference);
+
+        // Test loadFastaDictionary with File argument:
+        Assert.assertThrows(UserException.MalformedFile.class, () -> {
+            ReferenceUtils.loadFastaDictionary(testFastaFile);
+        });
+
+        // Test loadFastaDictionary with Stream argument:
+        try ( final ClosingAwareFileInputStream referenceDictionaryStream = new ClosingAwareFileInputStream(testFastaFile) ) {
+
+            Assert.assertThrows(UserException.MalformedFile.class, () -> {
+                ReferenceUtils.loadFastaDictionary(referenceDictionaryStream);
+            });
```

Comment:
You should be using TestNG's `expectedExceptions` feature here. Exceptions you expect to be thrown should go in the `@Test` annotation itself: `(expectedExceptions = UserException.MalformedFile.class)`. That way you don't have to call `assertThrows()` yourself. 

You'll need to break this up into two test cases to do this -- you can use a `DataProvider` with two test cases that passes in a `File` plus a boolean flag indicating whether to call the stream version.